nba_py/__init__.py

```diff
@@ -1,16 +1,30 @@
 from requests import get
-from datetime import datetime
+from datetime import datetime, timedelta
 from nba_py.constants import League
+import os
 
 HAS_PANDAS = True
 try:
     from pandas import DataFrame
 except ImportError:
     HAS_PANDAS = False
 
+HAS_REQUESTS_CACHE = True
```

Comment:
If I merge this PR it will overwrite the work @mwilliammyers has done and will show up on the blame as work that you've submitted. Specifically for these requests_cache and referrer changes.